lib/texter.rb

```diff
@@ -0,0 +1,48 @@
+require 'twilio-ruby'
+
+class Texter
+
+	attr_reader :processed_orders
+
+	def initialize(client=Twilio::REST::Client.new(ENV['TWILIO_ACCOUNT_SID'],ENV['TWILIO_AUTH_TOKEN']))
```

Comment:
You could use a constant to set the default - it would be a bit tidier than this long line.